torch/autograd/gradcheck.py

```diff
@@ -24,7 +24,8 @@
 
 
 class GradcheckError(RuntimeError):
-    r"""Error raised by :func:`gradcheck` and :func:`gradgradcheck`"""
+    r"""Error raised by `gradcheck` and `gradgradcheck` functions."""
```

Comment:
same here - can we please use :func:`gradcheck` and :func:`gradgradcheck`

Reply:
Sorry, I have corrected it now.